### DISCUSSION TITLE
Customtkinter won't load

### DISCUSSION THREAD

**Discussion by GaryC1954** [TomSchimansky/CustomTkinter#455] [General]
Getting this error message when launching customtkinter with just the single button example:

File "C:\Users\garyc\PycharmProjects\Excel_Project\venv\customtkinter.py", line 2, in <module>
    import customtkinter
  File "C:\Users\garyc\PycharmProjects\Excel_Project\venv\customtkinter.py", line 5, in <module>
    customtkinter.set_default_color_theme("blue")  # Themes: blue (default), dark-blue, green
AttributeError: partially initialized module 'customtkinter' has no attribute 'set_default_color_theme' (most likely due to a circular import)

I am new at this but have uninstalled customtkinter and reinstalled.  Also checked the install location compared to other pip installed apps.

**Comment by TomSchimansky** [comment]
Do you named your file `customtkinter.py `? I don't think you can name your .py file the same as the library you want to import. Your then importing your own .py file with nothing in there...

**Comment by GaryC1954** [comment]
While I did have the .py file saved as customtkinter.py, I changed the file name to cust.py but still getting a similar error message:

C:\Users\garyc\PycharmProjects\Custom\venv\Scripts\python.exe C:/Users/garyc/PycharmProjects/Custom/cust.py 
Traceback (most recent call last):
  File "C:\Users\garyc\PycharmProjects\Custom\cust.py", line 2, in <module>
    import customtkinter
ModuleNotFoundError: No module named 'customtkinter'

**Comment by GaryC1954** [comment]
I have found and fixed my problem.  I originally had installed Python 3.10.6 but after having problems installing several packages, found that it was recommended to use Anaconda, so I installed Anaconda, which downloaded Python 3.9.12.  I think this messed with my Pycharm path.  
I found on the internet that adding a path to Anaconda\Library\bin directory allowed me to download customtkinter (and others) using pip. 

Please close this what started out as an issue.  Thanks !!!
